src/test/rehydration.test.js

```diff
@@ -316,13 +319,15 @@ describe('rehydration', () => {
       `)
     })
 
-    it('should not change styles if rendered in the same order they were created with', () => {
-      injectGlobal`
+    it.skip('should not change styles if rendered in the same order they were created with', () => {
```

Comment:
This should still be there I guess to check the injection order. Let me know what you need to fix this up :) if you think another test already covers injection order, we can remove this

Reply:
So I was thinking that while `injectGlobal` would only inject styles once, `createGlobalStyle` will inject for each rendered component. Actually, I am a bit confused here.

Reply:
Ah I See, so it’s related to my later comment. Let’s delete this test for this PR then or keep it slipped with a TODO comment

Reply:
Yes. So these two tests seem to be failing massively and they are related to how `name` in general is handled or not-handled with local and global rules. So I will skip them and add comments that they should eventually work before we release.